server/src/internalClusterTest/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java

```diff
@@ -782,21 +782,24 @@ public Settings onNodeStopped(String nodeName) {
      * Tests shard recovery throttling on the target node. Node statistics should show throttling time on the target node, while no
      * throttling should be shown on the source node because the target will accept data more slowly than the source's throttling threshold.
      */
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/103204")
     public void testTargetThrottling() throws Exception {
         logger.info("--> starting node A with default settings");
-        final String nodeA = internalCluster().startNode();
+        final String nodeA = internalCluster().startNode(
+            Settings.builder()
+                .put(RecoverySettings.INDICES_RECOVERY_MAX_BYTES_PER_SEC_SETTING.getKey(), randomIntBetween(10, 200), ByteSizeUnit.MB)
+        );
 
         logger.info("--> creating index on node A");
-        ByteSizeValue shardSize = createAndPopulateIndex(INDEX_NAME, 1, SHARD_COUNT_1, REPLICA_COUNT_0).getShards()[0].getStats()
-            .getStore()
-            .size();
+        createAndPopulateIndex(INDEX_NAME, 1, SHARD_COUNT_1, REPLICA_COUNT_0);
 
-        long chunkSize = Math.max(1, shardSize.getBytes() / 10);
+        var nodeARecoverySettings = new RecoverySettings(
+            internalCluster().getInstance(Settings.class, nodeA),
+            clusterService().getClusterSettings()
+        );
+        // Use a fraction of the size that causes throttling to be triggered on the source node's RateLimiter
+        long chunkSize = nodeARecoverySettings.rateLimiter().getMinPauseCheckBytes() / 2;
```

Comment:
I think that we should populate the index with at least 2x the chunk size worth of data and that's not guaranteed right now?